### ISSUE TITLE
Config environments not working

### ISSUE THREAD

**Issue by donenvy** [ceesvanegmond/minify#66]
Hi in start.php if I return 'local' environment it ignores the minification, if I return 'production' produces minification which is fine.

What doesn't work is if I am in 'local' environment and decide I want to see what the minification looks like and remove 'local' from the ignore_environments, it still produces the raw files, doesn't take the settings into account.

Same visa versa, if I am in 'production' and decide I want to see what the raw files are, and specify in the config to ignore 'production' it still produces the minified version. 

Basically nothing I put in the ignore_environments works.

Only way I can make it work and switch between minified and not minified is by specifying 'local' or 'production' in the start.php. Config file does nothing?

**Comment by donenvy** [comment]
Ignore me, I've been making changes to the config file in the vendor package all this time. Just noticed that the actual config file to make changes to is in the app folder. My apologies closing this now
